schema_based_on_diagram.sql

```diff
@@ -0,0 +1,41 @@
+                               
+ CREATE DATABASE diagram_database;
+ 
+ \c diagram_database;
+
+ CREATE TABLE patients (
+ id INT NOT NULL PRIMARY KEY,
+ name VARCHAR(30),
+ date_of_birth DATE);
+
+ CREATE TABLE medical_histories(
+ id INT NOT NULL PRIMARY KEY,
+ admitted_at TIMESTAMP NOT NULL,
+ patient_id INT NOT NULL,
+ status VARCHAR(50),
+ FOREIGN KEY (patient_id) REFERENCES patients(id));
+
```

Comment:
- [ ] A table should be created between `medical_histories` and `treatments` since all "join table" for the many-to-many relationships is created even though it is not represented on the diagram. check this example.
`  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int references medical_histories(id),
    treatment_id int references treatments(id),
    );`.
The name of the table can be different. It depends on you.